frontend/src/services/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+const baseUrl = import.meta.env.VITE_BASE_URL;
```

Comment:
Could you update the readme to mention that you need this URL in env, and what it should default to for dev?

Reply:
yes